sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -567,6 +567,8 @@ class Analyzer(
           case u @ UnresolvedFunction(name, children, isDistinct) =>
             withPosition(u) {
               registry.lookupFunction(name, children) match {
+                // Do not wrap a WindowFunction in an AggregateExpression.
+                case wf: WindowFunction => wf
```

Comment:
nvm, AggregateWindowFunction could be WindowFunction and AggregateFunction in the same time.